docs/articles/documentation/reference/test-api/testcontroller/openwindow.md

```diff
@@ -6,7 +6,7 @@ permalink: /documentation/reference/test-api/testcontroller/openwindow.html
 
 # t.openWindow method
 
-Opens a new browser window.
+Opens a new browser window and returns the matching window object.
 
 ```js
 t.openWindow( url )
```

Comment:
we can show that it returns an object with an arrow:

`t.openWindow( url ) → Object`